BeVolt/Apps/Src/Voltage.c

```diff
@@ -4,12 +4,12 @@
  */
 
 #include "Voltage.h"
-#include "LTC6811.h"
+#include "LTC681x.h"
```

Comment:
Is there something you needed from LTC681x.h? The LTC6811 src file is a wrapper for the LTC681x src file so the LTC681x functions should not be used.

Reply:
No, I added it but forgot to remove it.